extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -44,12 +44,16 @@ amp-carousel[controls] .amp-carousel-button,
   left: 16px;
   background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="%23fff" viewBox="0 0 18 18"><path d="M15 8.25H5.87l4.19-4.19L9 3 3 9l6 6 1.06-1.06-4.19-4.19H15v-1.5z" /></svg>');
   background-size: 18px 18px;
+  opacity: 1;
+  visibility: visible;
```

Comment:
Sir, the fix is to solve the (forever) disappearing carousal buttons as one
cannot determine if it is a carousal or not. This fix does not hamper the
intended behaviour of making the nav button disappear incase of an end of
carousal(right nav) or incase of first carousal element(left nav) [tested].


Thanks

On Wed, Aug 17, 2016 at 2:40 PM, Sriram Krishnan <notifications@github.com>
wrote:

> In extensions/amp-carousel/0.1/amp-carousel.css
> <https://github.com/ampproject/amphtml/pull/4564#discussion_r75087575>:
>
> > @@ -44,12 +44,16 @@ amp-carousel[controls] .amp-carousel-button,
> >    left: 16px;
> >    background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="%23fff" viewBox="0 0 18 18"><path d="M15 8.25H5.87l4.19-4.19L9 3 3 9l6 6 1.06-1.06-4.19-4.19H15v-1.5z" /></svg>');
> >    background-size: 18px 18px;
> > +  opacity: 1;
> > +  visibility: visible;
>
> you can't do this as the behavior is intended. Please use the controls
> attribute of you want the controls to be permanently displayed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4564/files/376b04114bf09e979b8a90dfae55b4b1022f8a68#r75087575>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATuPGLQinMNJe_vHPzm6kx5BtHuK95M4ks5qgs_qgaJpZM4JmLIN>
> .
>

-- 
 

**************************************************************************************************************************************************************************************

*                                                      PropTiger.com **DISCLAIMER 
and CONFIDENTIALITY CAUTION*
**************************************************************************************************************************************************************************************

This email and any files transmitted with it are confidential and the 
Receiving Party shall use this Information exclusively only for the purpose 
of internal usage and not for other commercial and revenue generating 
purposes. The information contained in this mail may be subject to legal 
privilege and may be legally protected from Disclosures and the 
communication of which is prohibited. The Receiving Party shall treat the 
Confidential Information with at least the same degree of care and 
protection as it would use with respect to its own information. If you are 
not the intended recipient of this message or their agent, or if this 
message has been addressed to you in error, please immediately alert the 
sender by reply email and then destroy this message and any attachments.

Everything contained in this e-mail and attachments are relating to the 
official business of PropTiger.com. Communicating through email is not 
secure and capable of interception, corruption and delays. Anyone 
communicating with PropTiger.com by email accepts the risks involved and 
their consequences. The recipient should check this email and any 
attachments for the presence of viruses. PropTiger.com accepts no liability 
for any damage caused by any virus transmitted by this email.